### PR TITLE
ZK-4105: Combobox upper/lowercase to selected item trigger too late b…

### PR DESCRIPTION
**Pull request by scribetw**
…efore onChange

### CODE REVIEW

File: zul/src/archive/web/js/zul/inp/Combobox.js [hunk 1]
```diff
@@ -239,6 +239,10 @@ zul.inp.Combobox = zk.$extends(zul.inp.ComboWidget, {
 			//spec change (diff from zk 3): onSelect fired after onChange
 			//purpose: onSelect can retrieve the value correctly
 			//If we want to change this spec, we have to modify Combobox.java about _lastCkVal
+		} else {
+			// The value still didn't match any item, but onChange is still needed.
+			if (opts.sendOnChange)
```
Comment (DevChu) [inline zul/src/archive/web/js/zul/inp/Combobox.js]: -> else if

Comment (DevChu) [reply zul/src/archive/web/js/zul/inp/Combobox.js]: what about onChanging

**Comment by DevChu** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**